demosplan/DemosPlanCoreBundle/ResourceTypes/ProcedureTemplateResourceType.php

```diff
@@ -25,21 +27,31 @@
  * @property-read End $description
  * @property-read End $agencyMainEmailAddress
  * @property-read End $masterTemplate
+ * @property-read End $coordinate
+ * @property-read ProcedureMapSettingResourceType $mapSetting
  * @property-read AgencyEmailAddressResourceType $agencyExtraEmailAddresses
  * @property-read OrgaResourceType $orga               Do not expose! Alias usage only.
  * @property-read OrgaResourceType $owningOrganisation
  */
 final class ProcedureTemplateResourceType extends DplanResourceType
 {
+    /**
+     * @throws PathException
+     */
     protected function getProperties(): array
     {
-        return [
-            $this->createIdentifier()->readable()->sortable()->filterable(),
-            $this->createAttribute($this->agencyMainEmailAddress)->readable(true)->sortable()->filterable(),
-            $this->createAttribute($this->description)->readable()->aliasedPath($this->desc),
-            $this->createToManyRelationship($this->agencyExtraEmailAddresses)->readable()->filterable(),
-            $this->createToOneRelationship($this->owningOrganisation)->readable()->aliasedPath($this->orga)->sortable()->filterable(),
-        ];
+        if ($this->currentUser->hasAnyPermissions('area_public_participation', 'area_admin_map')) {
+            $properties[] = $this->createAttribute($this->coordinate)->readable()->aliasedPath(Paths::procedure()->settings->coordinate);
```

Comment:
I think rather than modifying the resource type file to include the coordinate, we should consume it from the existing resourcetype  where it's already referenced. What I mean is:

- The coordinate is an attribute belongs to the ProcedureMapSetting. That ResourceType is available when calling the line below: 
 ```
$properties[] = $this->createToOneRelationship($this->mapSetting)->aliasedPath(Paths::procedure()->settings)->readable();
        }
```
- I think the FE could potentially call the relationship, and get the coordinate from there. 
- then we can add the permission area_procedure_adjustments_general_location the permissions file in the needed project as `\demosplan\DemosPlanCoreBundle\ResourceConfigBuilder\ProcedureMapSettingResourceConfigBuilder::@$coordinate` requires that permission.

What do you think about this approach?

Reply:
The frontend actually already calls it via ProcedureMapSetting. There was a misunderstanding. I've removed the line in [6565401](https://github.com/demos-europe/demosplan-core/pull/3738/commits/6565401073294817e9ab301f16745213b3d851ec)

Reply:
ah, is the permission active for the coordinate in the needed project?

Reply:
No, I think it should be turned off for this project.